napari/_qt/qt_viewer.py

```diff
@@ -189,6 +193,21 @@ def __init__(self, viewer):
 
         self.setAcceptDrops(True)
 
+    def _create_performance_dock_widget(self):
+        """Create the dock widget that shows performance metrics.
+        """
+        if not perf.USE_PERFMON:
+            return None
+
+        return QtViewerDockWidget(
+            self,
+            QtPerformance(),
+            name='performance',
+            area='bottom',
+            allowed_areas=['bottom'],
```

Comment:
while playing with this a bit ... I kinda _wanted_ to put it on the right?  Any reason to restrict this here?

Reply:
I just picked bottom very early on. Seems to work fine on right so I changed it to allow either. We can change default to right if people want to?

Reply:
I think bottom is a good default.  I'd just remove the `allowed_areas` restriction

Reply:
Makes sense. I removed it.